eras/alonzo/test-suite/test/Test/Cardano/Ledger/Alonzo/Serialisation/Canonical.hs

```diff
@@ -77,13 +76,9 @@ checkCanonicalTerm = do
     TypeMapLen -> checkCanonicalMap
     TypeMapLen64 -> checkCanonicalMap
     TypeMapLenIndef -> fail "indefinite map encoding"
-    -- TypeTag ->
-    -- TypeTag64 ->
```

Comment:
normally I dislike commented out code, but perhaps this time it's a bit helpful? it's a reminder for what isn't implemented. but do as you like. :)

Reply:
`failNotImplemented` :tada:

Reply:
Fair enough. I did it the proper way. Commented out code is usually just a distraction, but it can be turned into something useful, such that compiler checked pattern match exhaustion check.